### PR TITLE
remove tmp code for ut

### LINKED ISSUE #135: Code Fixing need to done for isModuleEnabled() in db.go
The temporary change made in file db.go(pkg/common/dbm/db.go) needs to be fixed. The function isModuleEnabled() returns true in the beginning of the function.
dbgoerror

### PR DESCRIPTION
**Pull request by edisonxiang**
What type of PR is this?
/kind bug

What this PR does / why we need it:
* remove the hard code for unit test
* clean the temp file in unit test

Which issue(s) this PR fixes:
remove the hard code for unit test

Fixes #135

### CODE REVIEW

File: edge/pkg/common/dbm/db.go [hunk 1]
```diff
@@ -72,9 +73,22 @@ func InitDBManager() {
 	DBAccess.Using(dbName)
 }
 
+// Clean db file if it is neccessary
+func Clean() {
```
Comment (m1093782566) [inline edge/pkg/common/dbm/db.go]: CleanDBFile(file string) ??

Comment (edisonxiang) [reply edge/pkg/common/dbm/db.go]: the file name is defined in db.go, what about `CleanDBFile()`?

File: edge/pkg/common/dbm/db.go [hunk 2]
```diff
@@ -72,9 +73,22 @@ func InitDBManager() {
...
+	// Check if db file is existing
+	_, err := os.Stat(defaultDataSource)
```
Comment (m1093782566) [inline edge/pkg/common/dbm/db.go]: Can't we directly do `os.Remove()` and then handle the `*PathError`?

File: edge/pkg/common/dbm/db.go (hunk 1 above)
Comment (edisonxiang) [reply edge/pkg/common/dbm/db.go]: Done, Thanks

File: edge/pkg/common/dbm/db.go (hunk 2 above)
Comment (edisonxiang) [reply edge/pkg/common/dbm/db.go]: Done, Thanks

File: edge/pkg/common/dbm/db.go [hunk 3]
```diff
@@ -72,9 +73,20 @@ func InitDBManager() {
 	DBAccess.Using(dbName)
 }
 
+// CleanDBFile removes db file if it is neccessary
+func CleanDBFile() {
```
Comment (m1093782566) [inline edge/pkg/common/dbm/db.go]: This function is only used to remove `defaultDataSource` file?

Why not pass a file argument in?

File: edge/pkg/common/dbm/db.go [hunk 4]
```diff
@@ -72,9 +73,20 @@ func InitDBManager() {
 	DBAccess.Using(dbName)
 }
 
+// CleanDBFile removes db file if it is neccessary
```
Comment (sids-b) [inline edge/pkg/common/dbm/db.go]: nit: neccessary -> necessary 

This is causing CI failure.

File: edge/pkg/common/dbm/db.go (hunk 3 above)
Comment (edisonxiang) [reply edge/pkg/common/dbm/db.go]: I will Thanks

File: edge/pkg/common/dbm/db.go (hunk 4 above)
Comment (edisonxiang) [reply edge/pkg/common/dbm/db.go]: I will Thanks

**Pull request merged**